server/services/statementServiceTypes.ts

```diff
@@ -0,0 +1,18 @@
+type UserStatement = {
+  additionalComments?: Array<{ additionalComment: string; dateSubmitted: Date }>
+  id?: number
```

Comment:
The same thing applies regarding optionality here.. 

but one thing we could do is use the types defined in statementClientTypes to define a new type:

So the main goal is "we want to express a type defined as a statement but also with comments".

For this, rather than duplicating the existing structure we could use [intersection types](https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html).
```ts
import type { AdditionalComment, Statement } from '../data/statementsClientTypes'

type UserStatement = Statement & { additionalComments: AdditionalComment[] }

type Status = {
  value: string
  label: string
}
```

Might need to export the `statementClientType` members

And maybe `StatementWithComments` might be a better name?